src/lib/tls/tls_channel_impl.h

```diff
@@ -29,6 +29,17 @@ namespace TLS {
 class Client;
 class Server;
 
+enum class Record_Type: uint8_t {
```

Comment:
Unsure if this is the best place, but did not want to create another internal header just for this enum.